File: Week06/sentimental-mario-more/mario.py
from cs50 import get_int


# promt user for height of the pyramid
while True:
    h = get_int("Height: ")
    if 0 < h < 9:
        break
    else:
        print("The height should be between 1 and 8")

# build pyramid
for i in range(h):
    print((" " * (h - i - 1)) + ("#" * (i + 1)) + "  " + "#" * (i + 1))

File: Week01/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompt for input
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 | h > 8);

    // build pyramid  of blocks
    for (int i = 0; i < h; i++)
    {
        // left side of the pyramid
        for (int j = 0; j < h; j++)
        {
            if (j + i < h - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        // space in between the pyramid
        printf("  ");


        // right side of the pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}